docs/_data/navigation.yml

```diff
@@ -1,109 +1,38 @@
-      - page: Configuration
-        url: /docs/get_started/configuration.html
-      - page: How to Deploy
-        url: /docs/get_started/how-to-deploy/overview.html
-        entries:
-        - page: Using Conjur
-          url: /docs/get_started/how-to-deploy/using-conjur.html
-        - page: Using Kubernetes Secrets
-          url: /docs/get_started/how-to-deploy/using-kubernetes-secrets.html
+footer-docs:
+  - title: Why Secretless?
+    url: https://docs-staging.secretless.io/Latest/en/Content/Overview/WhySecretless.htm
+    target: target="_blank" rel="noopener noreferrer"
+  - title: How Does It Work?
+    url: https://docs-staging.secretless.io/Latest/en/Content/Overview/how_it_works.htm
+    target: target="_blank" rel="noopener noreferrer"
+  - title: Key Terms
+    url: https://docs-staging.secretless.io/Latest/en/Content/Overview/key_terms.htm
+    target: target="_blank" rel="noopener noreferrer"
   - title: Reference
-    url: /docs/reference.html
-    sections:
-      - page: Listeners
-        url: /docs/reference/listeners.html
-      - page: Handlers
-        url: /docs/reference/handlers/overview.html
-        entries:
-        - page: AWS - HTTPS
-          url: /docs/reference/handlers/http/aws.html
-        - page: Conjur - HTTP(S)
-          url: /docs/reference/handlers/http/conjur.html
-        - page: HTTP Basic Authentication
-          url: /docs/reference/handlers/http/basic.html
-        - page: MySQL
-          url: /docs/reference/handlers/mysql.html
-        - page: PostgreSQL
-          url: /docs/reference/handlers/postgres.html
-        - page: SSH Agent
-          url: /docs/reference/handlers/ssh_agent.html
-        - page: SSH
-          url: /docs/reference/handlers/ssh.html
-      - page: Credential Providers
-        url: /docs/reference/providers/overview.html
-        entries:
-        - page: CyberArk Conjur
-          url: /docs/reference/providers/conjur.html
-        - page: Environment
-          url: /docs/reference/providers/env.html
-        - page: File
-          url: /docs/reference/providers/file.html
-        - page: HashiCorp Vault
-          url: /docs/reference/providers/vault.html
-        - page: Keychain
-          url: /docs/reference/providers/keychain.html
-        - page: Literal
-          url: /docs/reference/providers/literal.html
-      - page: Configuration Managers
-        url: /docs/reference/config-managers/overview.html
-        entries:
-        - page: Configuration File
-          url: /docs/reference/config-managers/configfile.html
-        - page: Kubernetes CRD
-          url: /docs/reference/config-managers/k8s/crd.html
-      - page: Plugin Reference
-        url: /generated/pkg_secretless_plugin_v1.html
-      - page: Health Check
-        url: /docs/reference/health-check/overview.html
+    url: https://docs-staging.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm
+    target: target="_blank" rel="noopener noreferrer"
+
```

Comment:
Please add another link to the Docs footer list that reads "Plugin Reference" and points to https://secretless.io/generated/pkg_secretless_plugin_v1.html

Reply:
we haven't moved this over to the secretless docs site yet? If not, I will need to remove the sidebar (see below)
<img width="631" alt="screen shot 2019-01-31 at 11 43 33" src="https://user-images.githubusercontent.com/19418506/52045589-82e0f080-254d-11e9-9d20-f49440c38005.png">

Reply:
pulling from plugin/v1/doc.go

Reply:
This isn't going to move to the docs site - it's auto-generated with each build.

Reply:
so just leave the files in the codebase?

Reply:
the files will continue to get auto generated on build. we just need to link to them somewhere. please add a link to the Docs footer list that reads "Plugin Reference" and points to https://secretless.io/generated/pkg_secretless_plugin_v1.html

Reply:
This link has already been updated under `docs/_data/navigation.yml`.